do-catalog/src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,117 @@
+.checkbox {
+  position: relative;
+
+  .checkbox-decoration {
+    display: flex;
+    position: relative;
+    width: 24px;
+    height: 24px;
+    overflow: hidden;
+    border: 1px solid $checkbox__border-color;
+    border-radius: 2px;
+    pointer-events: none;
+  }
+
+  .checkbox-check {
+    transform-origin: 50% 50%;
+    stroke-width: 2;
+    stroke-dasharray: 48;
+    stroke-dashoffset: 48;
+    stroke: $checkbox-stroke__color;
+  }
+
+  .checkbox-decorationMedia {
+    display: flex;
+    position: relative;
+    width: 18px;
+    margin-right: auto;
+    margin-left: auto;
+    transform: translateY(3px);
+  }
+
+  .checkbox-input {
+    position: absolute;
+    -webkit-appearance: none;
+    appearance: none;
+    top: 0;
+    left: 0;
+    width: 24px;
+    height: 24px;
+    margin: 0;
+    padding: 0;
+    border-radius: 3px;
+    background: $checkbox__bg-color;
+    cursor: pointer;
+
+    &::before {
+      content: ' ';
```

Comment:
In general, there's no need to set a space for content.

Reply:
As this is imported from the dashboard, I prefer to leave it like this just in case it uses the space for sizing or whatsoever (I doubt it as it's an absolute and it sets the height and width, but just in case)